devscripts/__init__.py

```diff
@@ -1 +1 @@
-# Empty file needed to make devscripts.utils properly importable from outside
+# Empty file
```

Comment:
This should be removed. Empty `__init__` files are common and a comment saying `Empty file` doesn't help at all.